packages/react-mutation-mapper/src/component/oncokb/OncoKbCardBody.tsx

```diff
@@ -0,0 +1,122 @@
+
+const OncoKbMedicalDisclaimer = (
+    <p className={mainStyles.disclaimer}>
+        The information above is intended for research purposes only and should
+        not be used as a substitute for professional diagnosis and treatment.
+    </p>
+);
+
+export type OncoKbCardBodyProps = {
+    type: OncoKbCardDataType;
+    geneNotExist: boolean;
+    isCancerGene: boolean;
+    hugoSymbol: string;
+    pmidData: ICache;
+    indicator?: IndicatorQueryResp;
+    usingPublicOncoKbInstance: boolean;
+};
+
+@observer
+export default class OncoKbCardBody extends React.Component<
+    OncoKbCardBodyProps
+> {
+    @observable activeTab: 'oncogenicity' | 'mutationEffect' = 'oncogenicity';
+
+    getBody(indicator: IndicatorQueryResp) {
+        switch (this.props.type) {
+            case OncoKbCardDataType.TX:
+                return (
+                    <OncoKbCardTreatmentContent
+                        variant={indicator.query.alteration}
+                        oncogenicity={indicator.oncogenic}
+                        mutationEffect={indicator.mutationEffect.knownEffect}
+                        biologicalSummary={indicator.mutationEffect.description}
+                        mutationEffectCitations={
+                            indicator.mutationEffect.citations
+                        }
+                        geneSummary={indicator.geneSummary}
+                        variantSummary={indicator.variantSummary}
+                        tumorTypeSummary={indicator.tumorTypeSummary}
+                        treatments={indicator.treatments}
+                        pmidData={this.props.pmidData}
+                        usingPublicOncoKbInstance={
+                            this.props.usingPublicOncoKbInstance
+                        }
+                    />
+                );
+            default:
+                return <></>;
+        }
+    }
+
+    @computed get levelsOfEvidence() {
+        switch (this.props.type) {
+            case OncoKbCardDataType.TX:
+                return {
+                    levels: OncoKbHelper.LEVELS,
+                    levelDes: OncoKbHelper.LEVEL_DESC,
+                };
+            default:
+                return {
+                    levels: [],
+                    levelDes: {},
+                };
+        }
+    }
+
+    render() {
+        return (
+            <>
+                {!this.props.geneNotExist && (
+                    <div>
+                        {this.props.indicator &&
+                            this.getBody(this.props.indicator)}
+                        {this.props.usingPublicOncoKbInstance ? (
+                            <></>
+                        ) : (
+                            <>
+                                {/*Use tab pane style for the disclaimer to keep the consistency since the info is attached right under the tab pane*/}
+                                <div className={tabsStyles['tab-pane']}>
```

Comment:
@zhx828 @onursumer is this method of referecing styles (rather than dot notation) compliant with new typings?

Reply:
@alisman I maybe new on this topic. I think I use both types. Generally I tried to use a-b in non modules, but camel in modules. I didn't do much of the refactoring in this pr since the intension is to separate the logics. But I'm happy to do that. Do we have any guidance on which style should we use?

Reply:
@alisman good point, it is not complaint with the plugin introduced in #3192. But that's a webpack plugin and only works for the main package (`cbioportal-frontend`). We are still missing css typings for modular packages, because we build them with rollup. I'll open a separate ticket for that.

Reply:
Ok, so, @zhx828, when we introduce the typed css modules to packages, then we will need to bring these into compliance.   I believe that means dot notation and no dashes (styles.someClass).   I would just take opportunity now to adjust this here.

Reply:
@alisman sorry, I was wrong. Apparently both dot notation and brackets are valid, here is an example: https://github.com/cBioPortal/cbioportal-frontend/blob/9bf8a68e8ad0b36d75da3a3b47bd4eeeff3dd50e/src/pages/resultsView/download/DownloadTab.tsx#L677

This doesn't change the fact that we still need to introduce type checking for the packages. I have created a ticket for that: cBioPortal/cbioportal#7559